Source/WebCore/rendering/style/StyleRareNonInheritedData.h

```diff
@@ -249,6 +249,8 @@ class StyleRareNonInheritedData : public RefCounted<StyleRareNonInheritedData> {
 
     unsigned leadingTrim : 2; // LeadingTrim
 
+    OptionSet<MarginTrimType> marginTrim;
+
```

Comment:
Can you move this up so all the `unsigned` bits can be packed together?

Reply:
Yup! Done